### DISCUSSION TITLE
Postal worker how to increar what is the setup

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#2515] [Help with using Postal]
hi 

i am install postal i have 30 ip4 and 30 ipv6, email speed is to low,
  i have to config

workers:
  quantity: 20
  threads: 15

but worker postal status show 
4 worker can you please help me how to increase worker 

![image](https://github.com/postalserver/postal/assets/107031529/6e7aeb8d-c1b0-4416-b031-98318cba8429)

**Comment by willpower232** [comment]
You need to add more workers with the scale argument, have a look at https://github.com/postalserver/postal/discussions/1461#discussioncomment-1119403
